### ISSUE TITLE
Service workers, what?

### ISSUE THREAD

**Issue by gregorybarale** [Sfeir/bouffe-front#90]
# Titre

- Temps: _TBD_
- A partir de: _TBD_

## Description

Les services workers commencent à bien être supporté par tout le monde (sauf ie), une petite présentation?

**Comment by dahfazz** [comment]
Tu peux jeter un oeil aux slides de la school PWA, c’est open-bar

**Comment by gregorybarale** [comment]
Ah oui c'est vrai. Bon finalement n'est-ce pas redondant avec la Sfeir School?

**Comment by dahfazz** [comment]
Non. je n’ai pas vu les mêmes personnes au school et aux bouffes et c’est bien de focaliser sur le SW sur une prez plus courte.
